GoogleSignIn/Sources/Public/GoogleSignIn/GIDGoogleUser.h

```diff
@@ -40,6 +41,17 @@ NS_ASSUME_NONNULL_BEGIN
 /// The configuration that was used to sign in this user.
 @property(nonatomic, readonly) GIDConfiguration *configuration;
 
+/// The OAuth2 access token to access Google services.
+@property(nonatomic, readonly) GIDToken *accessToken;
+
+/// The OAuth2 refresh token to exchange for new access tokens.
+@property(nonatomic, readonly) GIDToken *refreshToken;
+
+/// An OpenID Connect ID token that identifies the user. Send this token to your server to
```

Comment:
We typically want doc comments to be a single sentence. You could consider adding a `note` for the second sentence. Look around for other examples in the repo and match that style.

Reply:
This is the style in `GoogleSignIn`. We can discuss that and use another PR to change them altogether.

Reply:
We're already using the style that I mentioned. 

https://github.com/google/GoogleSignIn-iOS/blob/main/GoogleSignIn/Sources/Public/GoogleSignIn/GIDSignInButton.h#L43